### PR TITLE
unp64 2.34 (new formula)

### PR DESCRIPTION
**Pull request by DennisOSRM**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/unp64.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Unp64 < Formula
+  desc "Generic C64 prg unpacker,"
+  homepage "http://iancoog.altervista.org"
+  url "http://iancoog.altervista.org/C/unp64_234.7z"
+  sha256 "86968afaa13b6c17fac7577041d5e3f3cc51cb534d818b5f360fddf41a05eaad"
```
Comment (fxcoudert) [inline Formula/unp64.rb]: `version "2.34"`

File: Formula/unp64.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  def install
+    system "make", "-C", "unp64_234/src", "unp64"
+    bin.install "unp64_234/src/Release/unp64"
+  end
+
+  test do
+    output = shell_output("#{bin}/unp64 2>&1", 1)
+    assert_match "UNP64 v2.34", output
```
Comment (fxcoudert) [inline Formula/unp64.rb]: We need a test that exercises the some of the functionality of the app.

File: Formula/unp64.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class Unp64 < Formula
+  desc "Generic C64 prg unpacker,"
+  homepage "http://iancoog.altervista.org"
+  url "http://iancoog.altervista.org/C/unp64_234.7z"
+  sha256 "86968afaa13b6c17fac7577041d5e3f3cc51cb534d818b5f360fddf41a05eaad"
+
+  def install
+    system "make", "-C", "unp64_234/src", "unp64"
+    bin.install "unp64_234/src/Release/unp64"
```
Comment (fxcoudert) [inline Formula/unp64.rb]: No `make install`? No docs, man page, or other stuff to install?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/unp64.rb (hunk 3 above)
Comment (DennisOSRM) [reply Formula/unp64.rb]: it's binary-only. Many of the tools from that particular community are kind of bare bones.

File: Formula/unp64.rb (hunk 1 above)
Comment (DennisOSRM) [reply Formula/unp64.rb]: done

**Comment by DennisOSRM** [comment]
@fxcoudert added a test case as requested

**Comment by DennisOSRM** [comment]
ping @fxcoudert

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @DennisOSRM for the pull request!

**Comment by fxcoudert** [comment]
Formula is merged, but the bottle gave me a weird error. Trying to bottle in a separate job.

**Comment by fxcoudert** [comment]
We are having CI trouble. I am reopening this to track the bottling.

**Comment by fxcoudert** [comment]
Bottled added in https://github.com/Homebrew/homebrew-core/commit/d1111b7c81e75bfc88f8e6edc8aa43e65d07ca0a

**Comment by DennisOSRM** [comment]
Thanks @fxcoudert for the help in getting this thru.

**Pull request closed without merging**